### PR TITLE
BaSP-TG-47: tasks - put and post

### PR DESCRIPTION
**Pull request by amilcarHernandez24**

### CODE REVIEW

**Review by disimoneleandro — DISMISSED** [review]
QA Approved!

**Comment by disimoneleandro** [comment]
QA Approved!

**Review by pabloschivazappa — DISMISSED** [review]
Nice work!

File: src/Components/Tasks/Form/index.js [hunk 1]
```diff
@@ -2,83 +2,56 @@ import { useEffect, useState } from 'react';
...
-      } else {
-        setChildren(() => editAndCreateMessage(data.message, data.data.description));
-      }
-      setModalDisplay(true);
-    } catch (error) {
-      setChildren(error);
-    }
+  const postTasks = () => {
+    dispatch(createTasks(descriptionValue));
+    console.log(descriptionValue);
```
Comment (juliancit0) [inline src/Components/Tasks/Form/index.js:39]:
```suggestion
```
You should delete this console log

**Review by juliancit0 — CHANGES_REQUESTED** [review]

File: src/Components/Tasks/Form/index.js (hunk 1 above)
Comment (amilcarHernandez24) [reply src/Components/Tasks/Form/index.js:39]: Thank juli, I'll do it.

**Review by juliancit0 — APPROVED** [review]

**Review by Juancapp — APPROVED** [review]
Oh yeah!

**Review by TomasBettini — APPROVED** [review]
Nice work!

**Pull request merged**